core/rawdb/accessors_rollup_event.go

```diff
@@ -144,3 +144,30 @@ func ReadFinalizedL2BlockNumber(db ethdb.Reader) *uint64 {
 	finalizedL2BlockNumber := number.Uint64()
 	return &finalizedL2BlockNumber
 }
+
+// WriteLastFinalizedBatchIndex stores the last finalized batch index in the database.
+func WriteLastFinalizedBatchIndex(db ethdb.KeyValueWriter, lastFinalizedBatchIndex uint64) {
```

Comment:
Could you add a simple unit test to cover these two read-write functions in `accessors_rollup_event_test.go`?

Reply:
no problem

Reply:
tests added